Sources/MongoSwift/MongoCollection+Write.swift

```diff
@@ -359,7 +359,7 @@ public struct DeleteOptions: Codable, BulkWriteOptionsConvertible {
 // Write command results structs
 
 /// The result of an `insertOne` command on a `MongoCollection`.
-public struct InsertOneResult: Decodable {
+public struct InsertOneResult: Codable {
```

Comment:
This was necessary because of the way I'm representing results as `BSON`s. It seemed relatively low risk to make this `Encodable` as well. 

We may run into some future situations where it is preferable to manually convert a result object to a `BSONDocument` and/or add the `Encodable` conformance for tests only.